include/gz/sim/rendering/WrenchVisualizer.hh

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef GZ_SIM_WRENCHVISUALIZER_HH_
+#define GZ_SIM_WRENCHVISUALIZER_HH_
+
+#include <memory>
+
+#include <gz/sim/config.hh>
+#include <gz/sim/rendering/Export.hh>
+#include <gz/utils/ImplPtr.hh>
+
+#include <gz/math/Vector3.hh>
+#include <gz/rendering/RenderTypes.hh>
```

Comment:
nit: include what you need and not all

Reply:
Since I'm using some declarations from those headers shouldn't I need to include them?